src/views/parents/parents.jsx

```diff
@@ -0,0 +1,128 @@
+                                <p><FormattedMessage id='parents.parentsSayingDescription' /></p>
+                                <p><b>"I just want to thank you all for making Scratch, and{' '}
+                                   for providing it for free.</b> My kids are doing amazing{' '}
+                                   things that they see as fun yet I know is educational,{' '}
+                                   valuable, and worthwhile. Thank you so much!!!"</p>
+                                <p><b>"My very shy but technical minded daughter has found{' '}
+                                   this to be a fantastic, safeoutlet for her creativity.</b>{' '}
+                                   She spends her free time creating ever more difficult{' '}
+                                   animations and sharing them with the scratch community.{' '}
+                                   The forums provide her with a group of like-minded{' '}
+                                   individuals with which she can hold on a conversation...{' '}
+                                   She now feels that computers, graphic design and{' '}
+                                   animation are something she would like to pursue in the{' '}
+                                   future. Your program has opened a whole new world to her{' '}
+                                   in so many ways, and I thank you wholeheartedly. "</p>
+                                <p><b>"My son is learning more than I can imagine from your{' '}
+                                   tool.</b> He is not a natural logical thinker but loves{' '}
+                                   LEGO. Your LEGO-like building block structure has moved him{' '}
+                                   forward by light-years in his logical thinking skills...He{' '}
+                                   can snap things together and begin to see the logic{' '}
+                                   reinforced by immediate feedback. Of course we work on{' '}
+                                   some things together--instant father and son time. This{' '}
+                                   is just incredible. Just a big thanks to you and MIT."</p>
+                            </li>
+                        </ul>
+                    </FlexRow>
+                </div>
+                
+                <div className="bodySide">
+                    <FlexRow>
+                        <Box title={''}>
+                            <li>
+                                <h3><FormattedMessage id='parents.learning' /></h3>
+                                <p><FormattedMessage id='parents.learningDescription' /></p>
+                            </li>
+
+                            <li>
+                                <h3><FormattedMessage id='parents.community' /></h3>
+                                <p><FormattedMessage id='parents.communityDescription' /></p>
```

Comment:
I'm seeing some links appear as escaped HTML here and in the Q&A too – I think that, for the JSON strings that have links in them, we'll need to use the `FormattedHTMLMessage` element for now. See below:
<img width="846" alt="screen shot 2016-10-17 at 3 38 07 pm" src="https://cloud.githubusercontent.com/assets/1815772/19453092/c4f96fa6-9481-11e6-8330-9150e9b0d991.png">